datasrc/network.py

```diff
@@ -417,6 +417,7 @@
 
 	NetMessage("Sv_TeamsState", []),
 
+	# deprecated, use showothers@netobj.ddnet.tw instead
```

Comment:
```suggestion
	# deprecated, use showothers2@netmsg.ddnet.tw instead
```

Reply:
Done.

Reply:
```suggestion
	# deprecated, use showothers@netmsg.ddnet.tw instead
```

Still the wrong name after the @.